server/manifest.go

```diff
@@ -19,8 +19,8 @@ const manifestStr = `
   "min_server_version": "5.26.0",
   "server": {
     "executables": {
-      "linux-amd64": "server/dist/plugin-linux-amd64",
       "darwin-amd64": "server/dist/plugin-darwin-amd64",
+      "linux-amd64": "server/dist/plugin-linux-amd64",
```

Comment:
Unrelated change?

Reply:
it is reordering, the result of some gofmt(?) op - should be innocuous(?). Do you think otherwise?

Reply:
Yes, it's definitely innocuous. Still, I would like to know why is this happening. `make apply` (which generates `server/manifest.go` generates the version with `linux-amd64` first.

Reply:
@hanzei - just checked - on my box the `make apply` (I guess invoked as part of `make dist`) is updating the order in the file.

Reply:
Oh, well the mysterious behaviors of go. Could you please fix the CI failure and we check if CI is fine with our change?

Reply:
The CI failure is because this PR depends of the server PR - which needs to go first. They should go away once the server PR is in.

Reply:
Why do you need to merge the server PR first? Can't you point to the server PR commit?

Reply:
I'd rather that we have the server PR first and then this PR. Anyway this PR will not be merged without the CI tests passing - so I'm not quite sure how the CI tests passing/not passing influence the dev CR approval(?)

Reply:
> so I'm not quite sure how the CI tests passing/not passing influence the dev CR approval

You will need to update `go.mod` to point to the newly added commit in https://github.com/mattermost/mattermost-server/pull/16769. That shouldn't require a re-review, right?

Reply:
Does it? Unless I miss smth, wouldn't the CI system ensure that the `go.mod` is the correct one? 
Validating that `go.mod` is updated - CI would not pass without it. 
Validating that the `go.mod` update is the correct one - the commit id is rather opaque to the user and CI would not pass without the right commit being pointed to?

Reply:
My approval is not tied to the `go.mod` changes but rather to CI passing in general. If CI doesn't pass, additional changes are required and hence I won't approve right now. Does that make sense to you? Happy to discus in a meeting.

Reply:
I guess I see the review process as smth independent of the CI passing. CI can fail for multiple reasons (test failure, linting/check-style failure, redux/server dependency, etc.) I have the trust in that the developer will fix those issues and and I'd not wait for him to fix them to approve their PR ( that would be like rubber-stamping the CI result - it is almost as if one reviewer would wait for another reviewer(Ci in this case)  approval before giving their own approval). Should one delay(or even tie their approval) to a lint failure issue? If the CI somehow detects some major issue(s) that impact the already reviewed code in  significant manner, then I trust that the developer (for his and everyone's sake) will submit it for re-review (that happens in my experience in very few of the cases).